jon-jc/dating-app-kr
# Implement Swiping System for Profile Cards with Clear Like and Dislik…

**Title:** Implement Swiping System for Profile Cards with Clear Like and Dislike Zones

**Description:**
This PR replaces the existing grid layout for displaying profile cards with a swiping system similar to Hinge. The swiping functionality is implemented using the `react-tinder-card` library, and Tailwind CSS is used for styling and animations. Clear indicators for "like" and "dislike" zones are added, and the cards can only be dragged when the mouse is pressed down.

**Changes:**
- Added `react-tinder-card` library for swiping functionality.
- Updated `Landing` component to use a swiping system instead of a grid layout.
- Applied Tailwind CSS classes for styling and smooth animations.
- Added visual indicators for "like" and "dislike" zones.
- Ensured cards can only be dragged when the mouse is pressed down.

**Screenshots:**
![image](https://github.com/jon-jc/dating-app-kr/assets/73568703/09165623-8bd3-4a2e-b881-19e793a07737)

**Testing:**
1. Navigate to the landing page.
2. Click and drag the profile cards to the left or right.
3. Verify that the cards swipe correctly and that the visual indicators for "like" and "dislike" zones are present.
4. Ensure the swiping animations are smooth and the profile card layout remains consistent.

**Additional Notes:**
- No custom CSS was used; all styles are implemented using Tailwind CSS.
- Ensure that the `react-tinder-card` library is installed as a dependency.

```tsx
"dependencies": {
  "react-tinder-card": "^1.3.1"
}
